src/main/java/nextstep/subway/fare/Fare.java

```diff
@@ -0,0 +1,66 @@
+package nextstep.subway.fare;
+
+import java.util.Objects;
+
+public class Fare {
+    private static final int BASIC_FARE = 1250;
+    private int value;
```

Comment:
```suggestion
    private static final int BASIC_FARE = 1250;
    
    private int value;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다.

Reply:
이런 부분까지 의견주셔서 정말 감사드려요!!
클래스 변수와 인스턴스 변수가 하나씩만 있어서 그대로 둔 걸로 기억하는데 이런 경우에도 공백을 추가하니 개인적으로 가독성이 더 좋다고 생각들었습니다!!